templates/corporate/case-studies/recurse-center-case-study.md

```diff
@@ -171,7 +171,7 @@ doesn’t see a message in some channel on that day, they’ll *never* see it.
 Without dedicating a lot of time, John is able to stay involved in the community
 on Zulip, and share his expertise where relevant. “I scan [recent
 topics](/help/recent-conversations) for places where I could help, and rely on
-email notifications for private messages,” John explains. The experience of
+email notifications for direct messages,” John explains. The experience of
```

Comment:
Since this is an attributed quote in a case study, let's leave it as 'private messages' for now.